### DISCUSSION TITLE
Upgrading prestashop 1.6 to 8, using the 1-click upgrade

### DISCUSSION THREAD

**Discussion by JoaoSantos12f** [PrestaShop/PrestaShop#35936] [Q&A]
Hi everyone, im kinda new with prestashop so i apologize in advance. So i have a shop in prestashop 1.6 and im trying to upgrade it to the latest version 8+ using the 1click upgrade free module, im doing it locally using xampp. Im trying to upgrade from 1.6 - 1.7 first and then 1.7 to 8 but Im facing a lot of issues such as:
- ajax/server error for action upgradeDb textstatus:"erro" errorThrown:" " jqXHR:" " 
I tried again and it went through, but now im stuck in login screen for backend like a loop.
Can someone guide me through the process of upgrading the store using 1click upgrade or is it actually not possible to use that module for the upgrade? (I am now trying to delete all the modules and themes and trying again)

**Comment by matks** [comment]
Hi @JoaoSantos12f you need to check into your PHP error logs to see what kind of issues you have.

The 1-click upgrade will take care of updating your database and your Core files. But it cannot guarantee what happens with your modules and theme as we don't know who built them.

> (I am now trying to delete all the modules and themes and trying again)

Sounds like a good strategy

Also read https://devdocs.prestashop-project.org/8/basics/keeping-up-to-date/use-autoupgrade-module/

**Reply by JoaoSantos12f** [reply]
Hi, thank you for the reply, i will check the php error logs and go from there.

**Reply by JoaoSantos12f** [reply]
Right now im using xampp with php 7.1 (which i belive is compatible with both prestashop 1.6 and 1.7), the process im trying is to update first to 1.7 with the 1click upgrade 4.14.3, and then to 8 with the latest version of the module, is that the right way to go for?

**Reply by matks** [reply]
Sorry for late answer, yes that is the correct way

ping @Quetzacoalt91 for your information ;)

**Comment by LaurynasLTU** [comment]
Upgrade from 1.6 to 1.7. and 8.0.5. (PHP 7.4) use 1-Click Direct Upgrade To 8.X
From 8.0.4 to 8.1 use 1-click upgrade - working perfect. (PHP 8.1)

Do not use 1-Click Direct Upgrade To 8.X module to upgrade 1.7 to 8.1.

8.0.5 presta with php 7.4 - many modules from 1.7, 8.x
8.1 presta with php 7.4 - a lot less modules
or
8.1 presta with php 8.1 - only new 1 month modules and many deprecated functions error messages, and bugs

90 percent of the presta 1.7, 8.0 versions of the module work with php 7.4 version, so now, at least for a year, there will be no point in using php 8.1 version. I'm using presta 8 with some modules that are for presta 1.7 and I'm glad they work. Although the downside is that with php 8.1, presta 8 works much faster, but more server resources are needed. 

**In general, presta 1.6 runs the fastest and uses the least amount of server resources.**
